### PR TITLE
[fix](inverted index)Add exception check when write bkd index

### PR DESCRIPTION
**Pull request by qidaye**
## Proposed changes

We are not catching the exception when add values in `bkd_writer`, if error throws, BE will run into segment fault.
So we add the exception check here to avoid coredump.

### CODE REVIEW

**Comment by qidaye** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

File: be/src/olap/rowset/segment_v2/inverted_index_writer.cpp [hunk 1]
```diff
@@ -520,23 +512,33 @@ class InvertedIndexColumnWriterImpl : public InvertedIndexColumnWriter {
...
+            std::string new_value;
+            size_t value_length = sizeof(CppType);
 
-        _value_key_coder->full_encode_ascending(&value, &new_value);
-        _bkd_writer->add((const uint8_t*)new_value.c_str(), value_length, _rid);
+            DBUG_EXECUTE_IF("InvertedIndexColumnWriterImpl::add_value_bkd_writer_add_throw_error", {
+                _CLTHROWA(CL_ERR_IllegalArgument, ("packedValue should be length=xxx"));
+            });
 
-        _rid++;
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/inverted_index_writer.cpp:539]: why delete `_rid++` ?

Comment (qidaye) [reply be/src/olap/rowset/segment_v2/inverted_index_writer.cpp:539]: In order to reuse `add_value` function in `add_array_values`, I moved the `_rid++` to the caller `add_numeric_values` in line 519.

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**